server/src/main/java/org/elasticsearch/snapshots/package-info.java

```diff
@@ -43,9 +43,9 @@
  * {@code UpdateIndexShardSnapshotStatusRequest}.</li>
  *
  * <li>If as a result of the received status update requests, all shards in the cluster state are in a completed state, i.e are marked as
- * either {@code SUCCESS}, {@code FAILED} or {@code MISSING}, the {@code SnapshotShardsService} will update the state of the {@code Entry}
+ * either {@code SUCCESS}, {@code FAILED} or {@code MISSING}, the {@code SnapshotsService} will update the state of the {@code Entry}
```

Comment:
I believe this was a mistake. @original-brownbear, could you please confirm that?